control/control.go

```diff
@@ -94,8 +88,8 @@ func (srv *HTTPServer) Run(bus *events.EventBus) {
 
 // Start sets up API routes with the event bus, listens on the control
 // socket, and serves the HTTP server.
-func (srv *HTTPServer) Start() {
-	endpoints := &Endpoints{srv.Bus}
+func (srv *HTTPServer) Start(bus *events.EventBus) {
+	endpoints := &Endpoints{bus}
```

Comment:
...but I'm wondering if we'll need to still treat HTTPServer as a publisher for `Endpoints` who publish events. Unsure how to handle reload through the control socket now as well, it's possible I'll need to add that back.